newrelic.yml

```diff
@@ -32,22 +32,6 @@ common: &default_settings
 # Defines the maximum number of request events reported from a single harvest.
 # analytics_events.max_samples_stored: 1200
 
-# If true, enables analytics event sampling.
-# analytics_events.enabled: true
-
-# If true, enables analytics event sampling.
-# analytics_events.enabled: true
-
-# Defines the maximum number of request events reported from a single harvest.
-# analytics_events.max_samples_stored: 1200
-
-# Defines the maximum number of request events reported from a single harvest.
-# analytics_events.max_samples_stored: 1200
-
-# Prefix of attributes to exclude from all destinations. Allows * as wildcard at
-# end.
-# attributes_exclude: []
-
```

Comment:
I'm not seeing this setting elsewhere in this file. Do you? If not, could you add it back?

Reply:
Sorry, I erased too much.
I added it back.

https://github.com/newrelic/newrelic-ruby-agent/pull/862/commits/73774e232f2d35d26e0033240beb61a34527429a